tools/util.wdl

```diff
@@ -301,12 +279,30 @@ task compression_integrity {
     runtime {
         memory: "~{memory_gb} GB"
         disk: "~{disk_size_gb} GB"
-        docker: 'quay.io/biocontainers/samtools:1.16.1--h6899075_1'
+        docker: 'quay.io/biocontainers/samtools:1.17--h00cdaf9_0'
         maxRetries: max_retries
     }
 }
 
 task add_to_bam_header {
+    # Should this be in samtools.wdl instead of util.wdl?
+    meta {
+        description: "Adds another line of text to the bottom of a BAM header"
+        outputs: {
+            updated_header: "The new header after modifications"  # TODO is this a lie? It would be missing a PG line from `reheader`. Why do we need this output?
+            header_lines: "Each line of the header is an entry in this array"  # Same question about needing this output
```

Comment:
Both are probably artifacts of development. It was easier to check the input file to the `samtools reheader` command when implementing. It would be missing a `@PG` record from the reheader, but I would consider that an artifact of the processing, rather than a meaningful component of the header update.

Reply:
A little unclear: it sounds like it would be safe to remove both these outputs?

Reply:
Yes, I think we're fine to remove these outputs.